README.md

```diff
@@ -49,6 +49,41 @@ Any other rules except these will be ignored.
 }
 ```
 
+### quality
+The `quality` phase can execute integration/functional tests that are appropriate for your cookbook. Currently, we support running [Test Kitchen](http://kitchen.ci) using the [kitchen-ec2 driver](https://github.com/test-kitchen/kitchen-ec2).
+
+In order to enable this functionality, perform the following prerequisite steps:
+
+* Add the following items to the appropriate data bag as specified in the [Handling Secrets](#handling-secrets-alpha) section 
+
+    *delivery-secrets <ent>-<org>-<project> encrypted data bag item*
+    ```json
+    {
+      "id": "<ent>-<org>-<project>",
+      "ec2": {
+        "access_key": "<ec2-access-key>",
+        "secret_key": "<ec2-private-key>",
+        "keypair_name": "<ec2-keypair-name>",
+        "private_key": "<JSON-compatible-ec2-keypair-private-key-content>"
+       }
+     }
+    ```
+    You can convert the private key content to a JSON-compatible string with a command like this: `ruby -e 'require "json"; puts File.read("<path-to-ec2-private-key>").to_json'`
+
+
+* Create a .kitchen-ec2.yml file in the root of your repo that has all of the required information needed by the [kitchen-ec2 driver](https://github.com/test-kitchen/kitchen-ec2) driver. delivery-truck will expose the following ENV variabls for use by kitchen:
```

Comment:
We have enabled kitchen before and the name convention we were following for this files is: `.kitchen.DRIVER.yml`

For this case that would be `.kitchen.ec2.yml` 

This is just to follow the name convention for other coming drivers. :smile: